model/context.go

```diff
@@ -50,17 +49,18 @@ func LoadContext(taskId, buildId, versionId, patchId, projectId string) (Context
 		if err != nil {
 			return *ctx, err
 		}
-
-		if ctx.ProjectRef != nil {
-			ctx.Project, err = FindProject("", ctx.ProjectRef)
-			if err != nil {
-				return *ctx, err
-			}
-		}
 	}
 	return *ctx, nil
 }
 
+// GetProject returns the project associated with the Context.
+func (ctx *Context) GetProject() (*Project, error) {
+	if ctx.ProjectRef != nil {
+		return FindProject("", ctx.ProjectRef)
```

Comment:
are there cases where we might call this more than once in a call stack? should we cache it?

Reply:
done